City-of-Helsinki/drupal-module-helfi-tpr
# UHF-1772 Missing integration overviews

Testing:
* If you don't already have a site for testing, create one:
  * `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  * cd hel-platform
  * make new
* Update the module to include changes: `composer require drupal/helfi_tpr:dev-UHF-1772-missing-integration-overviews`
* Run `drush updb`
* Make sure you have imported some "Errand service" and "Service channel" content. If not, run:
  * `drush migrate:import tpr_errand_service`
  * `drush migrate:import tpr_service_channel`
* As an anonymous user, go to the pages `/admin/content/integrations/tpr-errand-service` and `admin/content/integrations/tpr-service-channel`. You should **not** see any content.
* As an admin user (for testing purposes, use a user whose UID is **not** 1), go to the same pages and you **should see** the list of the migrated content.
  * Test that it's possible to publish multiple items using the bulk update functionality.

Also check the site specific PRs:
* [drupal-helfi/pull/168](https://github.com/City-of-Helsinki/drupal-helfi/pull/168)
* [drupal-helfi-sote/pull/55](https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/55)
* [drupal-helfi-kymp/pull/51](https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/51)
